data/tlg1181/tlg1181.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,783 @@
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+ <teiHeader xml:lang="eng">
+  <fileDesc>
+   <titleStmt>
+    <title>Aristarchus of Samos, the Ancient Copernicus: a History of Greek Astronomy to
+     Aristarchus, Together with Aristarchus's Treatise on the Sizes and Distances of the Sun and
+     Moon</title>
+    <author>Aristarchus</author>
+    <editor>Sir Thomas Heath</editor>
+    <sponsor>Harvard College Librart</sponsor>
+    <funder>Harvard Library Arcadia Fund</funder>
+    <principal>Gregory Crane</principal>
+    <respStmt>
+     <persName xml:id="DDD">Digital Divide Data</persName>
+     <resp>Corrected and encoded the text</resp>
+    </respStmt>
+    <respStmt>
+     <persName>Gregory Crane</persName>
+     <resp>Editor-in-Chief, Perseus Digital Library</resp>
+    </respStmt>
+    <respStmt>
+     <persName>Matt Munson</persName>
+     <resp>Project Manager (University of Leipzig), 2016 - present</resp>
+    </respStmt>
+    <respStmt>
+     <persName>Annette Gessner</persName>
+     <resp>Project Assistant (University of Leipzig) 2015 - present</resp>
+    </respStmt>
+    <respStmt>
+     <persName>Thibault Clérice</persName>
+     <resp>Lead Developer (University of Leipzig) 2015 - 2017</resp>
+    </respStmt>
+    <respStmt>
+     <persName>Bruce Robertson</persName>
+     <resp>Technical Advisor (Mount Allison University)</resp>
+    </respStmt>
+   </titleStmt>
+   <publicationStmt>
+    <authority>Harvard College Library</authority>
+    <idno type="tlg1181.tlg001.1st1K-grc.xml"/>
+    <availability>
+     <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International License</p>
+    </availability>
+    <date>2017</date>
+    <publisher>Harvard College Library</publisher>
+    <pubPlace>United States</pubPlace>
+   </publicationStmt>
+   <sourceDesc>
+    <listBibl>
+     <biblStruct>
+      <monogr>
+       <title>Aristarchus of Samos, the Ancient Copernicus: a History of Greek Astronomy to
+        Aristarchus, Together with Aristarchus's Treatise on the Sizes and Distances of the Sun and
+        Moon</title>
+       <editor>
+        <persName>
+         <name>Sir Thomas Heath</name>
+        </persName>
+       </editor>
+       <author ref="urn:cts:greekLit:tlg1181.tlg001">Aristarchus</author>
+       <imprint>
+        <publisher>Oxford</publisher>
+        <pubPlace>The Claredon Press</pubPlace>
+        <date>1913</date>
+       </imprint>
+       <biblScope unit="volume"/>
+      </monogr>
+      <ref target="https://archive.org/details/aristarchusofsam00heatuoft"/>
+     </biblStruct>
+    </listBibl>
+   </sourceDesc>
+  </fileDesc>
+  <encodingDesc>
+   <p>Text encoded in accordance with the latest EpiDoc standards</p>
+   <refsDecl n="CTS">
+    <cRefPattern matchPattern="(.+).(.+)" n="section"
+     replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:div[@n='$2'])"/>
```

Comment:
There is no second level of the citation scheme here, only "chapter"s. So the "section" cRefPattern should be deleted.